crates/core/src/plugin/mod.rs

```diff
@@ -195,6 +195,10 @@ pub enum ResolveKind {
   Entry,
   /// static import, e.g. `import a from './a'`
   Import,
+  /// static exports default
+  Default,
```

Comment:
The `ResolveKind` is used to distinguish the dependency type. For resolve we should create a new type instead of add unrelated types here